main/admin/settings.lib.php

```diff
@@ -1327,7 +1327,7 @@ function generateSettingsForm($settings, $settings_by_access_list)
                             get_lang($row['comment']),
                             get_lang('MB')
                         ),
-                        array('maxlength' => '8')
+                        array('maxlength' => '8', 'id' => $row['variable'])
```

Comment:
This is slightly dangerous too. I would rather build a string like 'id' => 'setting-'.$row['variable']
There are a few settings that can be added through plugins and it would be better to avoid any kind of weird circumstance where the ID conflicts with another basic element of the page.